go/libraries/doltcore/sqle/enginetest/dolt_queries_help.go

```diff
@@ -0,0 +1,231 @@
+		Expected: []sql.Row{},
+	},
+	{
+		Name:        "dolt_help names are correct",
+		SetUpScript: []string{},
+		Query:       "select name from dolt_help",
+		Expected: []sql.Row{
+			{"dolt_add"},
+			{"dolt_reset"},
+			{"dolt_clean"},
+			{"dolt_commit"},
+			{"dolt_branch"},
+			{"dolt_checkout"},
+			{"dolt_merge"},
+			{"dolt_conflicts_resolve"},
+			{"dolt_cherry_pick"},
+			{"dolt_revert"},
+			{"dolt_clone"},
+			{"dolt_fetch"},
+			{"dolt_pull"},
+			{"dolt_push"},
+			{"dolt_remote"},
+			{"dolt_backup"},
+			{"dolt_tag"},
+			{"dolt_gc"},
+			{"dolt_rebase"},
+		},
+	},
+	{
+		Name:        "dolt_help types are correct",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select type from dolt_help where name='dolt_rebase'",
+				Expected: []sql.Row{
+					{"procedure"},
+				},
+			},
+			{
+				Query: "select type from dolt_help where name='dolt_gc'",
+				Expected: []sql.Row{
+					{"procedure"},
+				},
+			},
+			{
+				Query: "select type from dolt_help where name='dolt_tag'",
+				Expected: []sql.Row{
+					{"procedure"},
+				},
+			},
+		},
+	},
+	{
+		Name:        "dolt_help synopses are correct",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select synopsis from dolt_help where name='dolt_conflicts_resolve'",
+				Expected: []sql.Row{
+					{"dolt conflicts resolve --ours|--theirs <table>..."},
+				},
+			},
+			{
+				Query: "select synopsis from dolt_help where name='dolt_merge'",
+				Expected: []sql.Row{
+					{`dolt merge [--squash] <branch>
+dolt merge --no-ff [-m message] <branch>
+dolt merge --abort`,
+					},
+				},
+			},
+			{
+				Query: "select synopsis from dolt_help where name='dolt_branch'",
+				Expected: []sql.Row{
+					{`dolt branch [--list] [-v] [-a] [-r]
+dolt branch [-f] <branchname> [<start-point>]
+dolt branch -m [-f] [<oldbranch>] <newbranch>
+dolt branch -c [-f] [<oldbranch>] <newbranch>
+dolt branch -d [-f] [-r] <branchname>...`},
+				},
+			},
+		},
+	},
+	{
+		Name:        "dolt_help short descriptions are correct",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select short_description from dolt_help where name='dolt_clean'",
+				Expected: []sql.Row{
+					{"Deletes untracked working tables"},
+				},
+			},
+			{
+				Query: "select short_description from dolt_help where name='dolt_checkout'",
+				Expected: []sql.Row{
+					{"Switch branches or restore working tree tables"},
+				},
+			},
+			{
+				Query: "select short_description from dolt_help where name='dolt_fetch'",
+				Expected: []sql.Row{
+					{"Download objects and refs from another repository"},
+				},
+			},
+		},
+	},
+
+	{
+		Name:        "dolt_help long descriptions are correct",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query: "select long_description from dolt_help where name='dolt_add'",
```

Comment:
This test is probably over specified -- we tweak these pretty often, don't want to check the whole thing.

I would probably do something like `select INSTR(long_description, "add") > 0`, the idea is just a spot test to be sure this returns the correct contents, not to check the contents verbatim.